docs/product/components/popovers.html

```diff
@@ -262,40 +260,227 @@ <h3 class="fs-body2 mb2">Teams</h3>
+                    <td><span class="s-label--status ml0">Default preventable</span> Fires immediately before showing and positioning the tooltip. This fires before the tooltip is first displayed to the user, and can be used to create or initialize the tooltip element. Calling <code class="stacks-code">.preventDefault()</code> cancels the display of the popover.</td>
+                </tr>
+                <tr>
+                    <th scope="row"><code class="stacks-code">s-tooltip:shown</code></th>
+                    <td>Controller element</td>
+                    <td>Fires immediately after showing the tooltip.</td>
+                </tr>
+                <tr>
+                    <th scope="row"><code class="stacks-code">s-tooltip:hide</code></th>
+                    <td>Controller element</td>
+                    <td><span class="s-label--status ml0">Default preventable</span> Fires immediately before hiding the tooltip.  Calling <code class="stacks-code">.preventDefault()</code> prevents the removal of the tooltip.</td>
+                </tr>
+                <tr>
+                    <th scope="row"><code class="stacks-code">s-tooltip:hidden</code></th>
+                    <td>Controller element</td>
+                    <td>Fires immediately after hiding the tooltip.</td>
+                </tr>
+            </tbody>
+        </table>
+    </div>
+    {% header h3 | Examples | tooltip-examples %}
+    <p class="stacks-copy">If the user doesn’t need to interact with the <em>contents</em> of the popover, it may be appropriate to only show it on hover. This will make popovers feel like a tooltip. To do so, we provide an alternative controller, <code class="stacks-code">s-tooltip</code>, that shows the tooltip only on hover.</p>
+
+    {% header h4 | Title attribute %}
+    <p class="stacks-copy">In the simple case where no markup is needed in the tooltip, the popover element can be omitted and automatically generated using the <code class="stacks-code">title</code> attribute.</p>
+
+    <div class="stacks-preview">
+{% highlight html %}
+<button class="s-btn" role="button"
+        title="…"
+        data-controller="s-tooltip"
+        data-s-tooltip-placement="bottom-start">
+    …
+</button>
+{% endhighlight %}
+        <div class="stacks-preview--example bg-black-025">
+            <button class="s-btn s-btn__filled" role="button"
+                    data-controller="s-tooltip"
+                    title="This popover shows content from the title attribute."
+                    data-s-tooltip-placement="bottom-start">
+                Hover tooltip popover
+            </button>
+        </div>
+    </div>
+
+    {% header h4 | JavaScript configuration %}
+    <p class="stacks-copy">In cases where the tooltip needs to display simple text or HTML, the popover can be configured using JavaScript.</p>
+    <div class="stacks-preview">
+{% highlight javascript %}
+Stacks.setTooltipHtml(el,
+    "Plain text tooltip",
```

Comment:
Do we even need this plain text tooltip example since we accept HTML and we killed `setTooltipText`?

Reply:
Up to you. I left it because I liked that it showcased that it doesn't *need* to be html, but if you think it is redundant, then I'm ok with ditching it.

Reply:
Resolved. Brian has added explicit separation of html/text content and the docs have been updated to reflect this.